docs/download.html

```diff
@@ -0,0 +1,120 @@
+  <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js" type="text/javascript"></script>
+  <script type="text/javascript">WebFont.load({  google: {    families: ["Open Sans:300,300italic,400,400italic,600,600italic,700,700italic,800,800italic","Roboto:300,regular,500"]  }});</script>
+  <!-- [if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js" type="text/javascript"></script><![endif] -->
+  <script type="text/javascript">!function(o,c){var n=c.documentElement,t=" w-mod-";n.className+=t+"js",("ontouchstart"in o||o.DocumentTouch&&c instanceof DocumentTouch)&&(n.className+=t+"touch")}(window,document);</script>
+  <link href="images/favicon.ico" rel="shortcut icon" type="image/x-icon">
+  <link href="images/webclip.png" rel="apple-touch-icon">
+</head>
+<body class="body-2">
+  <div data-collapse="medium" data-animation="default" data-duration="400" role="banner" class="navigation-bar w-nav">
+    <div class="w-container"><img src="images/iristrans.png" loading="lazy" width="56" sizes="56px" srcset="images/iristrans-p-800.png 800w, images/iristrans-p-1080.png 1080w, images/iristrans.png 1280w" alt="" class="image">
+      <a href="index.html" class="brand-link w-nav-brand">
+        <h1 class="brand-text">Iris Shaders</h1>
+      </a>
+      <nav role="navigation" class="navigation-menu w-nav-menu">
+        <a href="index.html" class="navigation-link w-nav-link">Home</a>
+        <link rel="prefetch" href="/">
+        <a href="about.html" class="navigation-link w-nav-link">Learn More</a>
+        <link rel="prefetch" href="/about">
+        <a href="download.html" aria-current="page" class="navigation-link-button w-nav-link w--current">Download Now</a>
+        <link rel="prerender" href="/download">
+      </nav>
+      <div class="hamburger-button w-nav-button">
+        <div class="w-icon-nav-menu"></div>
+      </div>
+    </div>
+  </div>
+  <div class="section">
+    <div class="container w-container">
+      <div class="section-title-group">
+        <h2 data-w-id="967e2b13-e58f-7cc7-bb5b-a7a1fa9f4792" style="opacity:0" class="section-heading centered">The Iris Installer</h2>
+        <div data-w-id="732748e7-4c98-18e7-ed34-efc539421d45" style="opacity:0" class="section-subheading center">designed to streamline your Iris experience</div>
+      </div>
+      <p data-w-id="63a984b5-6769-5dea-8742-d35bbde7c0d5" style="opacity:0" class="paragraph-10">Iris&#x27;s installer will automatically download and install Iris and Sodium for great graphics at smooth framerates. Simply run the installer, choose your version, and hit install. Don&#x27;t believe us? Try it for yourself.</p>
+    </div>
+    <a data-w-id="749e71ea-d065-d8b8-e64a-ed39ce24565f" style="opacity:0" href="#" class="big-button w-button">Download Now</a>
```

Comment:
Edit the `href` value on this line to add a download link for the iris installer. That is the only thing that needs to be done after merge but before site launch.